src/routes/indexRouter.ts

```diff
@@ -0,0 +1,148 @@
+import { Router, Request, Response } from "express";
+import { IUser, Users } from "../models/Users";
+import { hash } from "bcrypt";
+
+// Middleware
+import { checkAuthenticated } from "../middleware/checkAuthenticated";
+import { checkNotAuthenticated } from "../middleware/checkNotAuthenticated";
+
+// Utilities
+import { mail } from "../utilities/mailer";
```

Comment:
is this what's harassing me when i login????

Reply:
Yes.

Reply:
yessir